deafrica_waterbodies/waterbodies/polygons/make_polygons.py

```diff
@@ -0,0 +1,339 @@
+"""
+Make waterbody polygons from the Water Observations from Space all-time
+summary.
+
+Geoscience Australia - 2021
```

Comment:
Should we update this to note that we've made changes for Digital Earth Africa in 2023, but that it's based on the original from Geoscience Australia? @fangfy do we have other examples of how best to do attribution in other DE Africa repos?